packages/cli/tsconfig.json

```diff
@@ -10,5 +10,5 @@
     "noImplicitReturns": false
   },
   "include": ["src/**/*", "postcss.config.js"],
-  "exclude": ["node_modules", "src/starter-files/**", "src/playground/**", "**/*.test.ts"]
+  "exclude": ["node_modules", "src/starter-files/**", "src/playground/**", "**/*.test.ts", "src/workflows/**"]
```

Comment:
we can remove the workflow addition right?

Reply:
oh yeah